index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Home</title>
+  <link href="./styles/normalize.css" rel="stylesheet">
+  <link href="./styles/icono.min.css" rel="stylesheet" >
+  <link href="./styles/main.css" rel="stylesheet">
+  <script src="https://use.fontawesome.com/ed5b29bc68.js"></script>
+</head>
+
+<body>
+  <header>
+    <li><a href="index.html"><i class="icono-home"></i></a></li>
+    <nav>
+      <li><a href="portfolio.html"> My Pieces </a></li>
+      <li><a href="code-journal.html"> Learning </a></li>
+      <li><a href="about.html"> Resume </a></li>
+    </nav>
+  </header>
+  <section id="index-section">
```

Comment:
A `<main>` tag might be a better choice here than a `<section>`, since this is the core part of the page. In that case you would probably want to exclude the footer.